src/utils/AppDevUtils.ts

```diff
@@ -61,7 +61,13 @@ const validateURL = (url: string): boolean => {
 
 /* Sort matches by meetingTime in descending order */
 const sortMatchByMeetingTime = (x: Match, y: Match) => {
-  return y.meetingTime.getTime() - x.meetingTime.getTime();
+  const timeX = x.meetingTime;
+  const timeY = y.meetingTime;
+
+  if (!timeX) return 1;
+  if (!timeY) return -1;
+  if (!timeX && !timeY) return 0;
```

Comment:
I hope we don't run into cases where we're missing times - is this just something we need to account for when using sort or did you have a particular edge case in mind?

Reply:
It's just to be as robust as possible because we only want to run it on matches that have meeting times present. Since there are possible states where the meeting time hasn't been finalized, I wanted to explicitly write this out.